openqa-label-known-issues

```diff
@@ -188,4 +195,4 @@ label_issue() {
     investigate_issue "$testurl"
 }
 
-caller 0 > /dev/null || label_issue "$@"
+caller 0 > /dev/null || label_issue "$scheme://$host/$id"
```

Comment:
That is actually not a url to an openqa test...

Reply:
Yep, miss the `tests` string, but since this request changes is now takeover by #448 I'm not going to change this pr and just move into the new one!